### ISSUE TITLE
Update of the README

### ISSUE THREAD

**Issue by MBravoS** [MBravoS/splotch#74]
Following the work we have been doing to implement the [pyOpenSci suggestions](https://www.pyopensci.org/python-package-guide/documentation/repository-files/readme-file-best-practices.html) in SciCM, I think now is a good time to give a comprehensive update to the splotch README file. It will improve the usability and (visual appearance) of the package, so I think we should follow the same guidelines, regardless of whether we decide further down the line to submit this package to pyOpenSci/JOSS. I also think it makes sense to combine this issue #30 for our next release.

**Comment by AstroRobin** [comment]
Yes, agreed with lumping this together with Params. Especially because the usage of params probably needs it's own quite detailed documentation to be used effectively.
